nuxeo-jsf/nuxeo-platform-ui-web/src/main/java/org/nuxeo/ecm/platform/ui/web/restAPI/UploadRestlet.java

```diff
@@ -138,4 +117,11 @@ public void handle(Request req, Response res) {
         res.setEntity(rep);
     }
 
+    protected String addBinaryFileFromPlugin(Blob blob, DocumentModel folder) throws IOException {
+        FileManager fileManager = Framework.getService(FileManager.class);
+        DocumentModel doc = fileManager.createDocumentFromBlob(folder.getCoreSession(), blob, folder.getPathAsString(),
```

Comment:
looks like a duplicate of `PluginUploadRestlet.java`

Reply:
The goal of this commit was to migrate and test the restlets, not refactor them. They are deprecated anyway.